frontend/packages/dev-console/src/utils/pipeline-actions.tsx

```diff
@@ -40,21 +46,13 @@ export const getPipelineRunData = (pipeline: Pipeline, latestRun?: PipelineRun):
       pipelineRef: {
         name: pipeline.metadata.name,
       },
-      resources:
-        latestRun && latestRun.spec && latestRun.spec.resources
-          ? latestRun.spec.resources
-          : pipeline && pipeline.spec && pipeline.spec.resources
```

Comment:
that's why the above lines, if no 'latestrun' and no resources, this field would be []

Reply:
Sorry? Perhaps I am missing something - where are you looking into `pipeline.spec.resources` now? There are two fallbacks here... `latestRun.spec.resources` falls back into `pipeline.spec.resources` falls back into `[]`. You're not doing this anymore as far as I can see - you just do `latestRun` and `[]`.

Reply:
Yes, we should not be using 'pipeline.specs' any longer as it would simply not work and fail the validation. For a fallback a `[]` is used instead now.